### PR TITLE
Live update 2/2023

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of February's live update

### CODE REVIEW

File: content/news/public-oss-demo/2023/live-update-2-2023.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+tags: [contribute, event, community, live-update]
+---
+
+The project's live update was held for the second time this year and [streamed live](https://www.youtube.com/watch?v=XguCHCNlthA) on February 8th, 2023.
+
+## Demos that you don't want to miss!
+
+Safe to say, this was one of the best, if not the best, live updates we ever held. During February's event, we had a chance to see one of the most popular PrestaShop community projects. [Integration between PrestaShop and Vue Storefront](https://vuestorefront.io/blog/prestashop-integration-with-vue-storefront) created by [BinShops](https://www.binshops.com/prestashop-api).
+
+That was not all! The following demo about the [faceted search module](https://github.com/PrestaShop/ps_facetedsearch/releases/) presented by Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox)) was something that the community has wanted to see for a long time. Starting with version 3.11, you can filter products not only on the category pages but also on the new products, price drops, and bestsellers. There is also support for filtering search results which is now in the experimental phase.
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-2-2023.md]:
```suggestion
That was not all! The following demo about the [faceted search module](https://github.com/PrestaShop/ps_facetedsearch/releases/) presented by Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox)) was something the community wanted to see for a long time. Starting with version 3.11, you can filter products not only on the category pages but also on new products, price drops, and bestsellers. There is also support for filtering search results which is now in the experimental phase.
```

File: content/news/public-oss-demo/2023/live-update-2-2023.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Last but not least, a cherry on the cake! Overview of the new product page, which is coming to PrestaShop 8.1. It was created from scratch with hundreds of improvements, new features, and bug fixes. This new page addresses merchants' feedback: revamped combinations management, a new way of working with multi-store, and much more.
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-2-2023.md]:
```suggestion
Last but not least, the new product page overview, coming to PrestaShop 8.1, was like the cherry on top. It was created from scratch with hundreds of improvements, new features, and bug fixes. This new page addresses merchants' feedback: revamped combinations management, a new way of working with multi-store, and much more.
```

File: content/news/public-oss-demo/2023/live-update-2-2023.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+You can watch parts you're interested in using the links below.
+
+- [00:00](https://www.youtube.com/watch?v=XguCHCNlthA) Beginning of the session: the introduction
```
Comment (coffrep) [inline content/news/public-oss-demo/2023/live-update-2-2023.md]:
```suggestion
- [00:00](https://www.youtube.com/watch?v=XguCHCNlthA) Beginning of the session: introduction
```

**Review by coffrep — APPROVED** [review]
I'm approving with changes, let me know if there are any questions.

**Review by coffrep — APPROVED** [review]
Re-approving, just edited the "cherry on top" part.

**Comment by kpodemski** [comment]
thanks @coffrep :)

**Pull request merged**